README.md

```diff
@@ -43,11 +13,13 @@ The federation url has to be provided in `Config.js` file. The user will then be
 
 ```js
 const Config = {
-    global: "gdn1.prod.macrometa.io",
-    Toronto: "gdn1-tor1.prod.macrometa.io",
-    London: "gdn1-lon1.prod.macrometa.io",
-    Bengaluru: "gdn1-blr1.prod.macrometa.io",
-    Singapore: "gdn1-sgp1.prod.macrometa.io"
+  global: "gdn.paas.prod.macrometa.io",
```

Comment:
I think we need to change this to gdn.paas.macrometa.io

Reply:
yes, you're right. Please update it

Reply:
ok I have made the change